### PR TITLE
[luci] Add UNPACK in QuantizeWithMinMaxPass

### PR DESCRIPTION
**Pull request by hyunsik-yoon**
This makes QuantizeWithMinMaxPass process UNPACK op.

Signed-off-by: Hyun Sik Yoon <hyunsik.yoon@samsung.com>

### CODE REVIEW

**Comment by hyunsik-yoon** [comment]
This is added to enable TF for the following code:

```
rnn = RNN(GRUCell(.. , reset_after=True), return_sequences=True, return_state=True, ..)
```

**Comment by hyunsik-yoon** [comment]
This is added to enable TF for the following code:

```
rnn = RNN(GRUCell(.. , reset_after=True), return_sequences=True, return_state=True, ..)
```

Without this PR, error was 

<details>


```
$ gdb -q -cd ...../circle-quantizer/circle-quantizer --quantize_with_minmax float32 uint8 channel model.minmax_recorded.circle model.q.circle
....
(gdb) r
Starting program: ....... arch/circle-quantizer/circle-quantizer --quantize_with_minmax float32 uint8 channel model.minmax_recorded.circle model.q.circle

...

terminate called after throwing an instance of 'std::runtime_error'
  what():  Unsupported Op for const inputs

Program received signal SIGABRT, Aborted.
__GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
51      ../sysdeps/unix/sysv/linux/raise.c: No such file or directory.
(gdb) bt
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff57ea921 in __GI_abort () at abort.c:79
#2  0x00007ffff5e4d257 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#3  0x00007ffff5e58606 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#4  0x00007ffff5e58671 in std::terminate() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#5  0x00007ffff5e58905 in __cxa_throw () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#6  0x00007ffff704023e in luci::(anonymous namespace)::quantize_const_inputs (node=0x5555557f3be0, 
    output_type=loco::DataType::U8)
    at ........ deps/NNAS/compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:1121
...
(gdb) fr 6
#6  0x00007ffff704023e in luci::(anonymous namespace)::quantize_const_inputs (node=0x5555557f3be0, 
    output_type=loco::DataType::U8)
    at ........ deps/NNAS/compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:1121
1121              throw std::runtime_error("Unsupported Op for const inputs");
(gdb) p *node
$5 = (luci::CircleUnpack) {
  <luci::FixedArityNode<1, luci::CircleNodeImpl<(luci::CircleOpcode)108> >> = {
...
```

</details>

**Review by jinevening — APPROVED** [review]
LGTM

**Review by jinevening — CHANGES_REQUESTED** [review]
Please update verifier to check PACK and UNPACK.

Example (PAD): https://github.com/Samsung/ONE/pull/6514

**Comment by hyunsik-yoon** [comment]
@jinevening Verifying code was added. Could you take a look? Thanks.

File: compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp [hunk 1]
```diff
@@ -602,6 +602,19 @@ struct QuantizeSpecialActivation final : public luci::CircleNodeMutableVisitor<v
     set_act_qparam(node, i_scale, i_zp);
   }
 
+  void visit(luci::CircleUnpackOut *node)
+  {
+    auto unpack = loco::must_cast<luci::CircleUnpack *>(node->input());
+    auto input = loco::must_cast<luci::CircleNode *>(unpack->value());
+    auto i_qparam = input->quantparam();
+    assert(i_qparam->scale.size() == 1); // FIX_CALLER_UNLESS
```
Comment (seanshpark) [inline compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:610]: We don't need to check `i_qparam` is nullptr or not ?

Comment (hyunsik-yoon) [reply compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:612]: Added! Thanks!

**Review by jinevening — APPROVED** [review]
LGTM Thank you

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Pull request merged**